### PR TITLE
oVirt e2e: send status events to oVirt CI infrastructure

### PR DESCRIPTION
**Pull request by eslutsky**
for troubleshooting and reporting we need to send the CI job details,
this info will be used for correlating CI PR with oVirt Infra resources

Signed-off-by: Evgeny Slutsky <eslutsky@redhat.com>

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 1]
```diff
@@ -470,9 +495,17 @@ objects:
         export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
         update_image_registry &
 
+        #notify oVirt infrastucture that ocp  installation started
```
Comment (rgolangh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:498]: 2 spaces instead of 1 after `ocp`

**Comment by rgolangh** [comment]
@eslutsky can you prefix the PR with "WIP:" ?

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 2]
```diff
@@ -450,6 +452,29 @@ objects:
...
+
+
+          curl --insecure  \
+          --request POST \
+          --header "Accept: application/xml"  \
+          --header "Content-Type: application/xml" \
+          -u "${OVIRT_ENGINE_USERNAME}:${OVIRT_ENGINE_PASSWORD}" \
+          -d @/tmp/artifacts/installer/event.xml \
+          ${OVIRT_ENGINE_URL}/events
+
```
[the remark this answers is not in the document]
Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:476]: added `|| true` , so the error will be printed and ignored.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 3]
```diff
@@ -450,6 +452,29 @@ objects:
...
 
+        function send_event_to_ovirt(){
+
+        local install_state=$1
+        cat <<__EOF__ > /tmp/artifacts/installer/event.xml
+        <event>
+          <description>Openshift CI - cluster installation ; ${install_state} ; ${JOB_SPEC} ; ${OCP_CLUSTER} </description>
+          <severity>normal</severity>
+          <origin>openshift-ci</origin>
+          <custom_id>$(date +'%s')</custom_id>
```
[the remark this answers is not in the document]
Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: fixed by summing epoch seconds with  last 5 numbers from the build_id (Unique build number for each run. [0])

[0] https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

**Comment by eslutsky** [comment]
> @eslutsky can you prefix the PR with "WIP:" ?

not sure what you mean, we are not sending just the PR number,
we are sending the entire JOB_SPEC as JSON  in the description field.

**Comment by Gal-Zaidman** [comment]
/lgtm

A big help for ovirt CI infra

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (sdodson) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:488]: can we check the rehearsal to make sure these creds aren't logged?

**Comment by sdodson** [comment]
Checked the log at https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/10060/rehearse-10060-pull-ci-openshift-cluster-api-provider-ovirt-master-e2e-ovirt/1285453102056476672/artifacts/e2e-ovirt/container-logs/setup.log for cred leaks, I can see the event posted but no signs of creds in the logs, so lgtm

**Pull request merged**